src/cdk/testing/testbed/fake-events/type-in-element.ts

```diff
@@ -56,6 +57,12 @@ export function typeInElement(element: HTMLElement, ...modifiersAndKeys: any) {
           k.split('').map(c => ({keyCode: c.toUpperCase().charCodeAt(0), key: c})) : [k])
       .reduce((arr, k) => arr.concat(k), []);
 
+  // Do nothing if no keys have been specified. Calling this function with no keys should
+  // not result in a focus event being dispatched.
+  if (keys.length === 0) {
```

Comment:
@mmalerba I just want to bring up the discussion again. I'm still not quite sure what the right behavior is (in terms of focus). Somehow I could also see that the element receives focus.. but at the same time I don't see any good reason why someone would intentionally call `sendKeys` and expect the element to receive focus.

Reply:
I don't have a strong opinion, but it feels like a behavior people shouldn't depend on. We might as well not focus it so they don't try to depend on it.

Reply:
Yeah. I don't know what's best here. Maybe @jelbourn has a good idea.

Reply:
IMO the function should throw an error if you call `sendKeys` with no keys. That seems like it would be the least surprising no matter what.

Reply:
sgtm

Reply:
Agreed. Updated.